src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -1,41 +0,0 @@
-﻿using Microsoft.AspNetCore.Builder;
```

Comment:
I assume the only reason is when the options are used by services, not just middleware. Do we know for sure that these options are not used  (and won't be) in services?